cmd/immuclient/immuc/immuctest/currentstatus_test.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
To increase consistency of what you have done I suggest:
1. move all test files from immuctest in immuc.(one folder back)
2. in these file use immuc_test package Please see -> https://golang.org/pkg/cmd/go/internal/test/ 
> Cit: Test files that declare a package with the suffix "_test" will be compiled as a
> separate package, and then linked and run with the main test binary.

3. fix last imports

In this way we can have tests in the same folder of tested files (like in other places), and we can use test* folder as reported:

> The go tool will ignore a directory named "testdata", making it available
> to hold ancillary data needed by the tests.

In this way in my opinion is more consistent from what we have in the rest of the project.